lms/djangoapps/lti_provider/views.py

```diff
@@ -28,7 +31,7 @@
 REQUIRED_PARAMETERS = [
     'roles', 'context_id', 'oauth_version', 'oauth_consumer_key',
     'oauth_signature', 'oauth_signature_method', 'oauth_timestamp',
-    'oauth_nonce', 'user_id'
+    'oauth_nonce', 'user_id', 'resource_link_id', 'lti_version', 'lti_message_type'
```

Comment:
Why do we need this certification test? I really afraid that many of the existing customers will be affected by this change.
Another one question - why these parameters are required? I'm ok with `oauth_consumer_key` - because it is needed for authentication but I can't imagine why `lti_version`, `resource_link_id` or `lti_message_type == basic-lti-launch-request` are required.

I may suggest such solution: create new parameter "strict validation" for the LtiConsumer entry. It should be manageable through the Django admin in the "https://credocourseware.com/admin/lti_provider/lticonsumer/" section. By default it is False. If it is True - all validation above are applied.